flow-nodes/api-builder-plugin-format-date/test/test.js

```diff
@@ -192,37 +171,29 @@ describe('flow-node format-date', () => {
 	     * specified. As this is simply a wrapper node, we expect this to be allowed.
 		 */
 		it('should NOT fail to format date with invalid input format', async () => {
-			const flowNode = runtime.getFlowNode('format-date');
-
+			
 			const invalidFormat = 'clearly-not-a-format';
-			const result = await flowNode.formatDate({
+			const { value, output } = await flowNode.formatDate({
 				date: '2020-01-08T13:24:40+02:00',
 				format: invalidFormat
 			});
 
-			expect(result.output).to.equal('next');
+			expect(output).to.equal('next');
 			// Moments does some transformation based on its defined aliases. The result here is
-			const momentResult = 'c1/8/20203pmr1/8/2020y-not-pm-for24pmt';
-			expect(result.args).to.deep.equal([null, momentResult]);
-			expect(result.context).to.deep.equal({
-				date: momentResult
-			});
+			const momentResult = 'c1/8/20203pmr1/8/20202020-not-am-for0amt';
```

Comment:
another unexpected difference from the original

Reply:
Resolved.